### PR TITLE
feat: v-once support for v-if/else via a once flag

### PR DESCRIPTION
**Pull request by lulu0119**
Implemented basic v-once support for v-if/else via a **once** flag, please review and advise if any scenarios are unaddressed.

### CODE REVIEW

File: packages/runtime-vapor/src/apiCreateIf.ts [hunk 1]
```diff
@@ -10,6 +10,7 @@ export const createIf = (
   condition: () => any,
   b1: BlockFn,
   b2?: BlockFn,
+  once?: Boolean,
```
Comment (LittleSound) [inline packages/runtime-vapor/src/apiCreateIf.ts:13]:
```suggestion
  once?: boolean,
```

File: packages/compiler-vapor/src/generators/if.ts [hunk 2]
```diff
@@ -37,6 +37,7 @@ export function genIf(
       conditionExpr,
       positiveArg,
       negativeArg,
+      once ? String(once) : '',
```
Comment (LittleSound) [inline packages/compiler-vapor/src/generators/if.ts:40]:
```suggestion
      once && 'true',
```

**Review by LittleSound — CHANGES_REQUESTED** [review]

**Comment by lulu0119** [comment]
Thank you, sxzz, for helping me with the modifications.

**Review by LittleSound — APPROVED** [review]

**Review by sxzz — APPROVED** [review]

File: packages/compiler-vapor/src/generators/if.ts [hunk 3]
```diff
@@ -36,7 +36,8 @@ export function genIf(
       vaporHelper('createIf'),
       conditionExpr,
       positiveArg,
-      negativeArg,
+      negativeArg || (once ? 'null' : false),
```
Comment (sxzz) [inline packages/compiler-vapor/src/generators/if.ts:39]: Should add a placeholder if once is true but no negative branch.

**Pull request merged**